### ISSUE TITLE
Elasticsearch license not recognized

### ISSUE THREAD

**Issue by obudiman** [elastic/elasticsearch#20539]
**Elasticsearch version**: 5.0.0-alpha5

**Plugins installed**: x-pack

**JVM version**: java-1.8.0-openjdk-1.8.0.101-3.b13.24.amzn1.x86_64

**OS version**: Linux 4.4.11-23.53.amzn1.x86_64

**Description of the problem including expected versus actual behavior**:
We've got a new license file to extend the current license, but it doesn't seem to be recognized by ES. We got an empty response when trying to apply the license from the curl command.

`curl: (52) Empty reply from server`

The license content file looks just fine in the json format.

**Steps to reproduce**:
 1. Install new license using the following command:
> curl -u <user>:<password> -XPUT localhost:9200/_license -d @/tmp/license.json
 
 2. Verify the license using the following command
> curl -u <user>:<password> -XGET localhost:9200/_license

**Comment by jasontedor** [comment]
I suspect that you're running into #19834, you can workaround this by adding "-H Expect:" do the curl command line parameters.

**Comment by adamthedeveloper** [comment]
This is happening to me also. I've tried so many different suggestions across the net (which is a testament to how bad and poorly organized the elasticsearch documentation is). I'm in such a bad spot right now, I'm almost ready to find a different search indexing tool. I'm trying to debug some bad behavior on my production machines and would like to look at my log stashes to see what's happening, but alas, I'm stuck trying to debug how I can gain access to those logs. I'm going to open a separate issue on complete lockout in Kibana if the FREE licenses expire. Nag me to renew - but don't lock me out. This is rediculous.

**Comment by jasontedor** [comment]
@adamthedeveloper I'm sorry that you're running into trouble. The exact issue that was reported here was caused by a bug in a pre-release version of Elasticsearch 5.0.0, the problem does not exist in any production release of Elasticsearch. If you're running into an issue with the same symptoms, it is a different problem. Please open a topic on the [forum](https://discuss.elastic.co) in the X-Pack category (and feel free to link to it from here) and someone will try to help you.

**Comment by adamthedeveloper** [comment]
Thanks. https://discuss.elastic.co/t/free-license-expired-and-now-i-cant-do-anything/110229

I sure hope I can get someone to help me soon. I also tried IRC and not one there will reply. Looks like there are many active users there, but no one is paying any attention to new messages. I also tried calling the USA contact number - but just got voicemail on an operator extension. Very frustrated right now about this license thing. This should be much easier to deal with than it currently is.
